rrtmgp/mo_gas_optics_rrtmgp.F90

```diff
@@ -818,6 +822,8 @@ function load_int(this, available_gases, gas_names, key_species,  &
     real(wp),           dimension(:,:,:,:), intent(in   ) :: planck_frac
     real(wp),           dimension(:,:,:),   intent(in   ), &
                                               allocatable :: rayl_lower, rayl_upper
+    real(wp),           dimension(:,:),     intent(in   ), &
+                                              optional    :: optimal_angle_fit
```

Comment:
Is there a reason to make this optional? I don't think you've protected against the case where it isn't supplied